tests/lib/rule-stub.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * @fileoverview Tests for stub rules.
+ * @author Ian VanSchooten
+ */
+
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+var assert = require("chai").assert,
+    sinon = require("sinon"),
+    rulesStub = require("../../lib/rule-stub");
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+describe("rule-stub", function() {
+    var sandbox;
+
+    beforeEach(function() {
+        sandbox = sinon.sandbox.create();
```

Comment:
Hm, it isn't. I copied it from the eslint.js test without understanding it. I'll get rid of it.